New feature: Host Switcher (#1545)
theforeman/foreman-documentation: guides/common/modules/con_switching-between-hosts.adoc

> [id="con_switching-between-hosts_{context}"]

Reviewer: ```suggestion
[id="switching-between-hosts_{context}"]
```

Author: Addressed :+1:

= Switching between Hosts

When you are on a particular host in the {ProjectWebUI}, you can navigate between hosts without leaving the page by using the host switcher.

> Click the *⇄* next to the hostname.

Reviewer 2: ```suggestion
Click the *⇄* icon next to the hostname.
```

Author: Either is fine but I should go for original one considering minimalism and standard style guide.

Reviewer 2: In that case, I would do this:
```suggestion
Click *⇄* next to the hostname.
```

Author: :+1:

> 

Reviewer 2: ```suggestion
```

> 
> A list of hosts is displayed in alphabetical order with a pagination arrow and a search bar to find the host you are looking for.

Reviewer 2: ```suggestion
This displays a list of hosts in alphabetical order with a pagination arrow and a search bar to find the host you are looking for.
```
I don't think that this one sentence should be standalone. If you have a reason for this please say.

Author: Let's keep yours. It will give better coherence.
